src/pages/EditMonitor.vue

```diff
@@ -584,7 +593,6 @@ export default {
                     method: "GET",
                     interval: 60,
                     retryInterval: this.interval,
-                    databaseConnectionString: "Server=<hostname>,<port>;Database=<your database>;User Id=<your user id>;Password=<your password>;Encrypt=<true/false>;TrustServerCertificate=<Yes/No>;Connection Timeout=<int>",
```

Comment:
Should this have been deleted or should it just be set to `""`?

Reply:
I added it when I added the default value for the connection for sql server, but don't need it anymore. We don't need a placeholder value so it is ok removing.